package.json

```diff
@@ -7,7 +7,12 @@
     "test": "cypress run",
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "clean-report": "rm -r cypress/reports/*",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "pre-test": "npm run clean-report",
+    "post-test": "npm run combine:reports && npm run generate:reports"
```

Comment:
pre and post scripts should not have the `-` character. Right names should be:

- pretest
- posttest

In that way, when you run the test script those pre and post scripts will be executed automatically.

https://docs.npmjs.com/cli/v8/using-npm/scripts